src/wallet/coincontrol.h

```diff
@@ -17,6 +17,7 @@ class CCoinControl
 {
 public:
     CTxDestination destChange;
+    OutputType change_type;
```

Comment:
Probably should add some comments distinguishing use between this and destChange right above.

Had to look up why both are required. (this being useful for non-prescribed destination, but prescribed destination type only)

Reply:
Done.